src/main/java/racingcar/domain/RacingCars.java

```diff
@@ -16,11 +15,9 @@ public static RacingCars of(List<RacingCar> racingCars) {
     }
 
     public static RacingCars generateByNames(List<String> racingCarNames) {
-        List<RacingCar> racingCars = new ArrayList<>();
-        racingCarNames.forEach(racingCarName -> {
-            racingCars.add(new RacingCar(racingCarName));
-        });
-        return new RacingCars(racingCars);
+        return new RacingCars(racingCarNames.stream().map(racingCarName -> {
+            return new RacingCar(racingCarName);
+        }).collect(Collectors.toList()));
```

Comment:
```suggestion
        return new RacingCars(racingCarNames.stream()
                .map(RacingCar::new)
                .collect(Collectors.toList()));
```

람다 안에 `return` 은 불필요해보여요 '-'